lib/es5-helpers.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const ChildProcess = require('child_process')
+const Path = require('path')
+const FS = require('fs')
+
+let prefixPath = null
+
+function find(startDir, names) {
```

Comment:
Your helper might want to consider doing the same as: https://github.com/AtomLinter/linter-eslint/commit/4a9387f66ab6896a9525690cb9a3f4494831ae2b.  It fixes a bug I was seeing in the tests where the final path checked ended up being `.` instead of `/`.